tests/lowering/tensor_manipulation/test_stack.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import torch
+import ttnn
+import torch_ttnn
+
+
+class StackModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, tensors, dim):
+        return torch.stack(tensors, dim)
+
+
+@pytest.mark.parametrize(
+    "input_shape, dim",
+    [
+        ((12, 16), -1),
+    ],
```

Comment:
can we get a couple more corner cases captured here?

Reply:
ok, I will add more different cases